src/styles/getTooltipStyles.js

```diff
@@ -49,6 +49,30 @@ function computeHorizontalShift(windowWidth, xOffset, componentWidth, tooltipWid
     return 0;
 }
 
+/**
+ *
+ * @param {Number} [maxWidth] - Max-width for tooltip's wrapper.
+ * @param {Number} [tooltipTextWidth] -  Max-width for tooltip's wrapper.
+ * @returns {Object}
+ */
+function setWrapperWidth(maxWidth, tooltipTextWidth) {
+    if (!maxWidth) {
+        return ({});
+    }
+
+    if (tooltipTextWidth >= maxWidth) {
+        return ({maxWidth});
+    }
+
+    // Sum left and right wrapper padding, otherwise the wrapper will clip the text.
+    // Add 1 px safe buffer to avoid last word wrap or clip on special cases
+    // where the real text width is float but a little larger than ref.offsetWidth, which will return integer
+    // ex. real text width: 172.2 px, ref.offsetWidth: 172px
```

Comment:
This is not clear at all. What are you trying to say here? Why does 1px need to be added?